Failed resolution of: `Lkotlin/jvm/internal/Intrinsics`

## Description
<!-- Describe your issue -->
Launching the app fails with a `NoClassDefFoundError`:

```java
2025-03-14 23:40:36.457  4008-4008  ai.elimu.k...viewHelper ai.elimu.kukariri.debug              I  getIdsOfWordsPendingReview
2025-03-14 23:40:36.466  4008-4008  ai.elimu.k...ntActivity ai.elimu.kukariri.debug              I  idsOfWordsPendingReview.size(): 13
2025-03-14 23:40:36.475  4008-4008  AndroidRuntime          ai.elimu.kukariri.debug              D  Shutting down VM
2025-03-14 23:40:36.478  4008-4008  AndroidRuntime          ai.elimu.kukariri.debug              E  FATAL EXCEPTION: main
                                                                                                    Process: ai.elimu.kukariri.debug, PID: 4008
                                                                                                    java.lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/internal/Intrinsics;
                                                                                                    	at ai.elimu.content_provider.utils.ContentProviderUtil.getAllWordGsons(Unknown Source:2)
                                                                                                    	at ai.elimu.kukariri.assessment.WordAssessmentActivity.onCreate(WordAssessmentActivity.java:82)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:9002)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:8980)
                                                                                                    	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1545)
                                                                                                    	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4121)
                                                                                                    	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4326)
                                                                                                    	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:112)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:174)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:109)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:81)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2707)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:107)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:258)
                                                                                                    	at android.os.Looper.loop(Looper.java:356)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8836)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:620)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:966)
                                                                                                    Caused by: java.lang.ClassNotFoundException: Didn't find class "kotlin.jvm.internal.Intrinsics" on path: DexPathList[[zip file "/data/app/~~_hSpQLNdAb333zOjJx6kYA==/ai.elimu.kukariri.debug-c6UMp3UOUcYvzlMzkwb-6A==/base.apk"],nativeLibraryDirectories=[/data/app/~~_hSpQLNdAb333zOjJx6kYA==/ai.elimu.kukariri.debug-c6UMp3UOUcYvzlMzkwb-6A==/lib/arm64, /system/lib64, /system_ext/lib64]]
                                                                                                    	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
                                                                                                    	at java.lang.ClassLoader.loadClass(ClassLoader.java:637)
```

## Why?
<!-- Explain _why_ this issue is needed -->
This bug fix is needed since the app fails to open.

---

> elimu.ai's mission is to build innovative learning software that empowers out-of-school children to teach themselves basic reading📖, writing✍🏽 and math🔢 **within 6 months**.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -60,6 +60,7 @@ dependencies {
         exclude group: 'com.daasuu', module: 'EasingInterpolator'
     }
     implementation libs.easing.interpolator
+    implementation libs.kotlin.stdlib
 
     testImplementation libs.junit
 

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -10,6 +10,7 @@ constraintLayout = "2.1.3"
 appCompat = "1.4.1"
 androidXJunit = "1.1.3"
 androidXEspresso = "3.4.0"
+kotlin = "2.0.0"
 
 [libraries]
 elimu-model = { group = "com.github.elimu-ai", name = "model", version.ref = "elimuModel" }
@@ -19,6 +20,7 @@ easing-interpolator = { group = "com.github.MasayukiSuda", name = "EasingInterpo
 shine-button = { group = "com.sackcentury", name = "shinebutton", version.ref = "shineButton" }
 androidx-appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "appCompat" }
 androidx-constraint-layout = { group = "androidx.constraintlayout", name = "constraintlayout", version.ref = "constraintLayout" }
+kotlin-stdlib = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib", version.ref = "kotlin" }
 androidx-junit = { group = "androidx.test.ext", name = "junit", version.ref = "androidXJunit" }
 androidx-espresso = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "androidXEspresso" }
 

